syntax = "proto3";

package Graph;

message Edge {
    uint32 from = 1;
    uint32 to = 2;
    double weight = 3;
}

message Graph {
    repeated Edge edges = 1;
    uint32 vertex_count = 2;
}